cmd/create/cluster/cmd.go

```diff
@@ -1891,6 +1917,27 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	if etcdEncryption && isHostedCP && (etcdEncryptionKmsARN == "" || interactive.Enabled()) {
+		etcdEncryptionKmsARN, err = interactive.GetString(interactive.Input{
+			Question: "Etcd encryption KMS ARN",
+			Help:     cmd.Flags().Lookup("etcd-encryption-kms-arn").Usage,
+			Default:  etcdEncryptionKmsARN,
+			Required: etcdEncryption && isHostedCP,
```

Comment:
NP
If we entered the `if` statement in line `1920`, is it always going to equal `true`?